e2e-tests/commands/insurance/account/complete-sign-in-and-go-to-application.js

```diff
@@ -20,7 +20,7 @@ const completeSignInAndGoToApplication = (email = mockAccount[EMAIL]) => {
   completeInsuranceEligibilitySignInAndGoToDashboard(email);
 
   // go to the newly created application
-  insurancePages.dashboardPage.table.body.lastRow.referenceNumberLink().click();
+  insurancePages.dashboardPage.table.body.lastRow.submittedLink().click();
```

Comment:
this is going to clash with my PR so will need to be removed once mine is in - 1 application goes straight to it